romancal/pipeline/exposure_pipeline.py

```diff
@@ -73,6 +75,9 @@ def process(self, input):
         self.source_catalog.return_updated_model = True
         # make sure we update source catalog coordinates afer running TweakRegStep
         self.tweakreg.update_source_catalog_coordinates = True
+        # tweakreg can also save the L1 wcs files. However, ELP takes
+        # on this responsibiility, so disable the tweakreg saving.
+        self.tweakreg.save_l1_wcs = False
```

Comment:
```suggestion
        # tweakreg should save the L1 wcs files
        self.tweakreg.save_l1_wcs = True
```

We've had different conversations about this, but if we make this the default we'll do the right thing in operations.